### PR TITLE
Remove function that exactly matches parent class.

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Removes override function that matches parent

Before
----------------------------------------
Same function exists in child & parent

After
----------------------------------------
I moved it over onto  the parent class fn & apart from minor comment block differences it matches. 

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Core/Form/Task.php [hunk 1]
```diff
@@ -167,7 +167,7 @@ public function setContactIDs() {
   }
 
   /**
-   * Simple shell that derived classes can call to add buttons to
+   * Simple shell that derived classes can call to add buttons to.
    * the form with a customized title for the main Submit
```
Comment (colemanw) [inline CRM/Core/Form/Task.php]: I know it's not very good English, but the comment here appears to be (attempting) a complete sentence:

> Simple shell that derived classes can call to add buttons to the form

Makes even less sense when you add the full stop

> Simple shell that derived classes can call to add buttons to. the form

Comment (colemanw) [reply CRM/Core/Form/Task.php]: How about:

    /**
     * Adds a pair of buttons to the form.

Comment (eileenmcnaughton) [reply CRM/Core/Form/Task.php]: @colemanw fixed

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by colemanw** [comment]
retest this please

**Pull request merged**